docs/langref/src/concepts/layouting.md

```diff
@@ -1,28 +1,29 @@
 # Positioning and Layout of Elements
 
-All visual elements are shown in a window. Their position is stored in the `x` and `y`
-properties as coordinates relative to their parent element. The absolute position of an element
-in a window is calculated by adding the parent's position to the element's position. If the
-parent has a grandparent element, then that one is added as well. This calculation continues until
-the top-level element is reached.
+All visual elements are shown in a window. The `x` and `y` properties store
+the elements coordinates relative to their parent element. You calculate the
+absolute position of an element in a window by adding the parent's position to
+the element's position. If the parent has a grandparent element, then that one
+is added as well. This calculation continues until the top-level element
+is reached.
```

Comment:
Why did you rephrase it as "You calculate"? This implies that the developer has some kind of obligation. I prefer the passive form.

Another way would be to write "Slint determines the absolute position of an element by ..."